src/main/java/it/rate/webapp/dtos/EmailMessageDTO.java

```diff
@@ -1,3 +1,12 @@
 package it.rate.webapp.dtos;
 
-public record EmailMessageDTO(String to, String subject, String text) {}
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import org.hibernate.validator.constraints.Length;
+
+public record EmailMessageDTO(
+        @NotBlank(message = "Email cannot be empty") @Email(message = "Invalid email address") String from,
+        @NotBlank(message = "Email cannot be empty") @Email(message = "Invalid email address") String to,

```

Comment:
do we need to specify "to"?

It should always be us right?

Reply:
Not if we want to send email notifications to users in the future.